cmd/generator/main.go

```diff
@@ -246,6 +255,57 @@ func getReleasePlugin(ctx context.Context, client *github.Client, repositoryName
 	return plugin, nil
 }
 
+func downloadSignatures(assets []github.ReleaseAsset) ([]*model.PluginSignatures, error) {
+	signatures := make([]*model.PluginSignatures, 0, len(assets))
+	for _, asset := range assets {
+		hash, err := getHashFromSignatureFileName(asset)
+		if err != nil {
+			return nil, err
+		}
+		sig, err := getSignatureFromSignatureFileName(asset)
+		if err != nil {
+			return nil, err
+		}
+
+		signature := &model.PluginSignatures{
+			PublicKeyHash: hash,
+			Signature:     sig,
+		}
+		signatures = append(signatures, signature)
+	}
+	return signatures, nil
+}
+
+func getHashFromSignatureFileName(asset github.ReleaseAsset) (string, error) {
+	name := asset.GetName()
+	if !strings.HasSuffix(name, ".sig") && !strings.HasSuffix(name, ".asc") {
+		return "", errors.New("signature file has wrong extension")
+	}
+	name = name[:len(name)-4]
+	lastIndex := strings.LastIndex(name, "-")
+	if lastIndex == -1 {
+		return "", errors.New("can't find public key hash in the signature file name")
+	}
+	return name[lastIndex+1:], nil
+}
+
+func getSignatureFromSignatureFileName(asset github.ReleaseAsset) (string, error) {
```

Comment:
nit: this doesn't actually get the signature from the /filename/, but from the asset. Perhaps just call this `getSignatureFromAsset`?